simapp/simd/cmd/testnet.go

```diff
@@ -40,20 +40,55 @@ var (
 	flagOutputDir         = "output-dir"
 	flagNodeDaemonHome    = "node-daemon-home"
 	flagStartingIPAddress = "starting-ip-address"
+	flagEnableLogging     = "enable-logging"
+	flagGRPCAddress       = "grpc.address"
+	flagRPCAddress        = "rpc.address"
+	flagAPIAddress        = "api.address"
+	flagPrintMnemonic     = "print-mnemonic"
 )
 
+func addTestnetFlagsToCmd(cmd *cobra.Command) {
+	cmd.Flags().Int(flagNumValidators, 4, "Number of validators to initialize the testnet with")
+	cmd.Flags().StringP(flagOutputDir, "o", "./.testnets", "Directory to store initialization data for the testnet")
```

Comment:
Hmm... you raise a good point. This would require some rethinking of how `make localnet-start` builds these init files though. As currently the makefile expects the files created by `simd testnets init-files` to be able to be written directly to `./build` as opposed to something like `./build/chain-id-foobar`. 

I'll investigate how to get that sorted tomorrow- probably will involve some small tweaks to the makefile.  It does seems pretty weird behavior to have chain-id subdirectories be used only for `start` but not for `init-files`...

Reply:
Yeah, I also like the idea of storing in a chain-id subdirectory!

Reply:
We discussed this on a call and decided using chain-id subdirectories would be a nice improvement but it complicates the `make localnet-start` command in a way that is not ideal. We decided to leave this as is for now.